tests/data/makeOldWcsFitsFiles.py

```diff
@@ -0,0 +1,154 @@
+"""Construct two binary FITS tables containing lsst.afw.image.TanWcs
+
+oldTanWcs.fits is a pure TAN WCS
+oldTanSipWcs.fits is a TAN-SIP WCS
+"""
+from lsst.daf.base import PropertyList
+from lsst.afw.image import makeWcs
+
+
+def makeAndWriteWcs():
+    tanWcsMetadata = getTanWcsMetata()
+    tanWcs = makeWcs(tanWcsMetadata)
+    tanWcs.writeFits("oldTanWcs.fits")
+
+    tanSipWcsMetata = getTanSipWcsMetata()
+    tanSipWcs = makeWcs(tanSipWcsMetata)
+    tanSipWcs.writeFits("oldTanSipWcs.fits")
+
+
+def getTanWcsMetata():
+    metadata = PropertyList()
+    # the following was fit using CreateWcsWithSip from meas_astrom
```

Comment:
Was this really created with "CreateWcsWithSip" since there are no Sip terms?

Reply:
Good catch. The comment belonged to `getTanSipWcsMetata`. I've cleaned that up.